src/components/alert/alert.twig

```diff
@@ -10,7 +10,12 @@
   {% endblock %}
 {% endset %}
 
-<{{ tag_name }} class="c-alert{% if floating %} c-alert--floating{% endif %}{% if class %} {{ class }}{% endif %}">
+<{{ tag_name }}
+  {% if id %}id="{{id}}"{% endif %}
+  class="c-alert{% if floating %} c-alert--floating{% endif %}{% if class %} {{ class }}{% endif %}"
+  {% if hidden %}hidden{% endif %}
+  role="status"
```

Comment:
Should we make this configurable in case there _is_ an alert that justifies a `role` of `alert`?

```suggestion
  role="{{role|default('status')}}"
```

(If this opens up a can of worms, feel free to disregard for now... we can always improve on this stuff later.)

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/1914/commits/6786560b16f8494915ac9952c226218b437c3ba8